.github/workflows/ci.yml

```diff
@@ -23,6 +23,10 @@ jobs:
           node-version: 16
       - name: 📦 Install dependencies
         run: yarn
+      - name: 🛠️ Build lib
+        run: yarn libbuild
+      - name: 🔧 Fix eslint problems
+        run: yarn fix
```

Comment:
why?

Reply:
`yarn libbuild` - builds `share` directory
`yarn fix` - fixes semicolons created by libbuild

Reply:
Without `libbuild` share import is not recognized

Reply:
> Without `libbuild` share import is not recognized

Which makes eslint run in to errors

Reply:
In the newest commit, `eslint fix` was replaced with `eslint lint` so I think it resolves this conversation, as I explained the other steps.